activemodel/test/cases/translation_test.rb

```diff
@@ -126,4 +126,14 @@ def test_human_attribute_name_does_not_modify_options
     Person.human_attribute_name("gender", options)
     assert_equal({ default: "Cool gender" }, options)
   end
+
+  def test_raise_on_missing_translations
+    ActiveModel::Translation.raise_on_missing_translations = true
+
+    assert_raises I18n::MissingTranslationData do
+      Person.human_attribute_name("name")
+    end
+  ensure
+    ActiveModel::Translation.raise_on_missing_translations = false
+  end
```

Comment:
You could simplify this test using `Object#with`.

```suggestion
  require "active_support/core_ext/object/with" # at top of file
  
  def test_raise_on_missing_translations
    ActiveModel::Translation.with(raise_on_missing_translations: true) do
      assert_raises I18n::MissingTranslationData do
        Person.human_attribute_name("name")
      end
    end
  end
```

Reply:
Thank you!